src/Routes/RosPage/RosPage.js

```diff
@@ -26,25 +52,50 @@ class RosPage extends React.Component {
         super(props);
 
         this.state = {
-            page: 1,
-            perPage: 10
+            perPage: 10,
+            orderBy: 'display_name',
+            orderDirection: SortByDirection.asc,
+            columns: [
+                { key: 'display_name', title: 'Name', renderFunc: systemName },
+                { key: 'display_performance_score.cpu_score', title: 'CPU score', renderFunc: scoreProgress },
+                { key: 'display_performance_score.memory_score', title: 'Memory score', renderFunc: scoreProgress },
+                { key: 'display_performance_score.io_score', title: 'I/O score', renderFunc: scoreProgress },
+                { key: 'recommendation_count', title: 'Recommendations',
+                    renderFunc: recommendations, props: { isStatic: true } },
+                { key: 'state', title: 'State', props: { isStatic: true } }
+            ]
         };
 
+        this.inventory = React.createRef();
+        this.fetchSystems = this.fetchSystems.bind(this);
     }
 
-    async componentDidMount() {
-        await window.insights.chrome.auth.getUser();
```

Comment:
@karelhala, I have removed `getUser()` call. Please confirm if it is expected after adding changes with `InventoryTable` component.

Reply:
You don't need to do that on render, however you should call this before calling the `fetch` for ROS systems.